config/prod/test-redhatlinux.yaml

```diff
@@ -0,0 +1,21 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2.yaml
+stack_name: amazon-linux2
+parameters:
+  # The Sage deparment for this resource
+  Department: "CompOnc"
+  # The Sage project this resource will be used for
+  Project: "Amazon linux 2 base image"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+  # EC2 instance type (available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.micro"
+  AMIId: "ami-02219c267315128db"
```

Comment:
i don't think redhat will work with `managed-ec2-v2.yaml` template due to installation of aws-cfn-bootstrap package.  https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/templates/managed-ec2-v2.yaml#L500


it seems like it's slightly different to install on redhat.  https://forums.aws.amazon.com/thread.jspa?threadID=226199

Reply:
Hmmm... I don't use this distribution of linux ever, but just thought I would try it out.  I think if we just limited people to work with the amazon linux distributions / ubuntu, we don't need to include the redhat linux.  Thoughts?